views/welcome-page.twig

```diff
@@ -25,65 +25,196 @@
-    <div class="jumbotron">
-        <h1>WIP <small>{% trans "Welcome" %}</small></h1>
-        <p>{% trans "We'd like to take you on a short tour of the CMS using the tutorials listed below to get you up and running in no time. If you already know your way around, please skip." %}</p>
-    </div>
-    </div>
+        .tour-title {
+            font-size: 18px;
+            padding: 16px 16px 0px 16px;
+        }
+
+        .tour-image {
+            width: 100%;
+            border-bottom: 6px solid #30426a;
+        }
+
+        .tour-text {
+            line-height: 18px;
+            padding: 6px 16px 16px 16px;
+            letter-spacing: 0.25px;
+        }
+
+        .tour-icon {
+            position: absolute;
+            opacity: 0.9;
+            right: 16px;
+            top: 0;
+            padding: 10px 12px;
+            background-color: #333;
+            color: #f9f9f9;
+            border-radius: 0 0 6px 6px;
+            font-size: 18px;
+        }
+
+        /* running */
+        .tour-card.running .tour-image {
+            border-color: #e99d1a;
+        }
+
+        .tour-card.running .tour-icon {
+            background-color: #e99d1a;
+        }
+
+        /* done */
+        .tour-card.done .tour-image {
+            border-color: #5cb85c;
+        }
+
+        .tour-card.done .tour-icon {
+            background-color: #5cb85c;
+        }
+
+        .welcome-title {
+            font-size: 20px;
+            margin: 20px 0 20px 0;
+        }
 
-    <div>
-        <div class="col-xs-4">
-            <button type="button" class="btn btn-block btn-info btn-tour" data-tour-name="mainTour">Main Tour&nbsp;</button>
+        /* player icons */
+        .welcome-players-section .player-icon {
+            width: 60px;
+            height: 60px;
+            padding: 12px;
+            background-color: #337ab7;
+            border-radius: 10px;
+            display: block;
+            margin: 0 auto;
+        }
+
+        .player-title {
+            margin-top: 8px;
+        }
+
+        .tour-cards-section, .welcome-players-section {
+            display: table;
+            width: 100%;
+            max-width: 1200px;
+            margin-bottom: 10px;
+            margin-right: auto !important;
+            margin-left: auto !important;
+        }
+
+        .tour-card-container {
+            margin-bottom: 20px;
+        }
+
+        .tour-link {
+            cursor: pointer;
+            border-collapse: separate; /* IE fix */
+            box-shadow: 0px 0px 10px 1px #00000026;
+            opacity: 0.9;
+        }
+
+        .tour-link:hover {
+            opacity: 1;
+            box-shadow: 0px 0px 10px 3px #00000026;
+        }
+    </style>
+{% endblock %}
+
+{% block pageContent %}
+
+    <div class="container-fluid welcome-container">
+        <div class="col-xs-12">
+            <h2>{{ "Welcome to the [[PRODUCTNAME]] CMS"|trans|replace({'[[PRODUCTNAME]]': theme.getThemeConfig('theme_title') }) }}</h2>
```

Comment:
I think we can do this with `trans` substitutions instead.

```
{% set productName = theme.getThemeConfig('theme_title') %}
{% trans %}Welcome to the %productName% CMS{% endtrans %}
```

Reply:
@dasgarner , using the %variable% inside the trans element doesn't really seem to work for the replacement, but {{variable}} does, is it the latter the right one?